tests/r2-delayed-delete.unit.test.ts

```
@@ -0,0 +1,63 @@
+import type { S3Client } from '@aws-sdk/client-s3'
+import { describe, expect, it, vi } from 'vitest'
+import { buildDelayedDeleteKey, moveR2ObjectsToDelayedDelete, R2_DELAYED_DELETE_PREFIX } from '../scripts/r2_delayed_delete.ts'
+
+describe('R2 delayed delete', () => {
```

ESLint

[error] `describe`s should begin with lowercase

(test/prefer-lowercase-title)